### DISCUSSION TITLE
Documentation idea: the 3 ways to interaction with the database: Core, ORM, and raw sql

### DISCUSSION THREAD

**Discussion by taigit** [sqlalchemy/sqlalchemy#9473] [Usage Questions] [answered]
Core and ORM are very clearly documented.

But executing raw sql is buried in the docs (as far as I can tell).   Yet, it's a common case for auxiliary tooling code.
It would be helpful for me if there was a dedicated section for it:

How to use raw sql in SQLAlchemy? 

In Core: https://docs.sqlalchemy.org/en/20/core/connections.html#basic-usage

In ORM: https://docs.sqlalchemy.org/en/20/orm/persistence_techniques.html#using-sql-expressions-with-sessions

**Comment by zzzeek** [comment, marked as answer]
hi -

not sure if you've read through the most important parts of SQLAlchemy's documentation which are the [unified tutorial](https://docs.sqlalchemy.org/en/20/tutorial/index.html) and the [ORM Querying guide](https://docs.sqlalchemy.org/en/20/orm/queryguide/index.html)

`text()` is used as the lead example when SQL execution is introduced:

https://docs.sqlalchemy.org/en/20/tutorial/dbapi_transactions.html#basics-of-statement-execution

the query guide has a full section on using textual SQL with the ORM:

https://docs.sqlalchemy.org/en/20/orm/queryguide/select.html#getting-orm-results-from-textual-statements
